src/config.rs

```diff
@@ -6,6 +6,7 @@ pub struct Config {
     pub fonts: Vec<fpm::Font>,
     pub dependencies: Vec<fpm::Dependency>,
     pub ignored: ignore::overrides::Override,
+    pub is_translation_package: bool,
```

Comment:
At least keep track of what it is translation of na, why `bool`? `translation_of: Option<Package>` is my preferred type here.

Reply:
Changed my mind, should be `translation_of: Option<String>` since the original package could be on another repository, so original must be specified as a `dependency` and as `translation_of` both.

Reply:
Or maybe if they share a repo we skip `dependency` and only required `translation_of`.

Reply:
I just realised it is on `Config`, it is totally wrong. `is_translation` is a property of `fpm::Package`.

Reply:
In which case it looks like you need a helper and not data. I prefer you convert it to a method on `Config` if you want or just delete it.